### PR TITLE
add account to credit check

### PR DESCRIPTION
**Pull request by darmhoo**
Add  account name and number to credit check report

### CODE REVIEW

File: src/components/modals/ViewCreditReportModal.vue [hunk 1]
```diff
@@ -57,6 +57,18 @@
...
+        <li class="list-group-item my-3">
+          <label for="exampleFormControlSelect2">Account Number</label>
+          <p class="display-text">
+            {{ creditReport.accountNo }}
+          </p>
+        </li>
+        <li class="list-group-item my-3">
+          <label for="exampleFormControlSelect2">Bank</label>
+          <p class="display-text">
+            {{ creditReport.bankName }}
```
Comment (teebarg) [inline src/components/modals/ViewCreditReportModal.vue:69]: Can we have a Bank list? so we could be consistent with the name.

Comment (darmhoo) [reply src/components/modals/ViewCreditReportModal.vue:69]: We dont have an updated bank api

Comment (darmhoo) [reply src/components/modals/ViewCreditReportModal.vue:69]: How should i go about it

Comment (teebarg) [reply src/components/modals/ViewCreditReportModal.vue:69]: Just a list on the frontend

Comment (darmhoo) [reply src/components/modals/ViewCreditReportModal.vue:69]: Done

**Review by AltaraNg — APPROVED** [review]

**Pull request merged**